ifrs17/Import/2ImportScope-PresentValue.ipynb

```diff
@@ -96,7 +96,9 @@
             "source": [
                 "## Interest and Discount Rates and Factors",
                 "\n",
-                "\nThe factors used for discounting have the same granularity as the cash flow, i.e. monthly. The yield curves have yearly granularity, so the annual Interest factor is 1 + interest rate. The monthly Interest Interest and Discount factors are obtained from the annual factors such that the product of 12 months results in the annual factors, as follows:",
+                "\nCalculation of present values can be performed using Current or Locked-in rates (locked at the inception year of the data node). During calculation, this is controlled by the Economic Basis which is either set to C for Current rates or L for Locked-in rates. In addition, the Economic Basis N identifies absence of discounting, corresponding to Nominal values. In this case an empty array is returned for both Interest and Discount values. ",
+                "\n",
+                "\n<br> The factors used for discounting have the same granularity as the cash flow, i.e. monthly. The yield curves have yearly granularity, so the annual Interest factor is 1 + interest rate. The monthly Interest Interest and Discount factors are obtained from the annual factors such that the product of 12 months results in the annual factors, as follows:",
```

Comment:
maybe this should be latex?

Reply:
I have improved the text here. There is already some latex for the formulas. I did not add formulas for the dependency on EconomicBasis.